src/input_common/udp/client.h

```diff
@@ -28,9 +30,32 @@ struct PortInfo;
 struct Version;
 } // namespace Response
 
+enum class PadMotion {
+    GyrX,
```

Comment:
```suggestion
    GyroX,
```

etc. We don't need to mince words here.